### DISCUSSION TITLE
Bug Report: Running the Game

### DISCUSSION THREAD

**Discussion by ThisIsVortexx** [KadeArchive/Kade-Engine#401] [Q&A]
Whever you try to run the game through cmd by using "lime test windows" the game doesnt run and only gives me a bunch of errors in the code that i dont know how to fix, those code errors are these:

Desktop/Kade/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
Desktop/Kade/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access
source/PlayState.hx:258: characters 7-24 : Warning : Std.is is deprecated. Use Std.isOfType instead.
source/PlayState.hx:478: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
Desktop/Kade/haxe/lib/hscript/2,4,0/hscript/Interp.hx:145: characters 8-18 : Warning : Std.is is deprecated. Use Std.isOfType instead.
Desktop/Kade/haxe/lib/newgrounds/1,1,4/lib/Source/io/newgrounds/components/ScoreBoardComponent.hx:55: characters 24-44 : Warning : Std.is is deprecated. Use Std.isOfType instead.
Desktop/Kade/haxe/lib/polymod/git/polymod/format/ParseRules.hx:635: characters 7-32 : Warning : Std.is is deprecated. Use Std.isOfType instead.
Desktop\Kade\Kade-Engine-1.4.2\export\release\windows\obj>setlocal enabledelayedexpansion
Error: 64bit is not automatically supported for this version of VC. Set HXCPP_MSVC_CUSTOM and manually configure the executable, library and include paths
Error: Error: 64bit is not automatically supported for this version of VC. Set HXCPP_MSVC_CUSTOM and manually configure the executable, library and include paths

Is there any way i could fix these?

**Comment by isakube** [comment]
All of these are deprecations or version issues. Give the results of `haxe` and `haxelib list`. daniel will figure out anything else because i'm not a professional

You should have Haxe 4.1.5, flixel 4.8.1 (my version, it works fine), lime 7.9.0, and uhh y'know what just rerun the commands in building.md i have an assignment due in -2 days so i don't even know why i'm here
hope i helped at least a little bit

**Comment by ThisIsVortexx** [comment]
> All of these are deprecations or version issues. Give the results of `haxe` and `haxelib list`. daniel will figure out anything else because i'm not a professional
> 
> You should have Haxe 4.1.5, flixel 4.8.1 (my version, it works fine), lime 7.9.0, and uhh y'know what just rerun the commands in building.md i have an assignment due in -2 days so i don't even know why i'm here
> hope i helped at least a little bit

Okay so yes i have everything installed but i just recently started so i dont know if build.md is a program or something so im still getting the same running error

**Comment by puyoxyz** [comment]
building.md is a file that you read.

you don't have all the components of visual studio that you need i think? open the installer again and install the components listed in [BUILDING.md](https://github.com/KadeDev/Kade-Engine/blob/master/BUILDING.md) (below the windows dependencies section)
